lux/action/custom.py

```diff
@@ -42,4 +43,20 @@ def custom(ldf):
         vis.score = interestingness(vis,ldf)
     # ldf.clear_intent()
     vlist.sort(remove_invalid=True)
-    return recommendation
\ No newline at end of file
+    return recommendation
+
+def custom_actions(ldf):
+    if (lux.actions.__len__() > 0):
+        recommendations = []
+        for action_name in lux.actions.__dir__():
+            validator = lux.actions.__getattr__(action_name).validator 
```

Comment:
What is the difference between `_get_action`, ` __getactions__`, and `__getattr__(action_name)`?

Reply:
__getactions__ returns a list of actions. _get_action is unused so I will probably delete it